src/controllers/__tests__/item.test.js

```diff
@@ -351,4 +339,58 @@ describe('Items Endpoints', () => {
       expect(responseBody.message).toBe('Requested item ID is not valid!');
     });
   });
+
+  describe('PUT /api/items/:id', () => {
+    test('Should update name on matching item', async () => {
+      const response = await request(server)
+        .put(`/api/items/${itemId}`)
+        .send({ name: 'Couch' });
+
+      expect(response.header['content-type']).toContain('application/json');
+      expect(response.statusCode).toBe(200);
+      expect(response.body.name).toEqual('Couch');
+    });
+
+    test('Should update isAvailable field on matching item', async () => {
```

Comment:
A better test statement would be: `Should set matching item as not available`

Try to avoid technical implementation language (such as variable names) on your test statements and keep it simple English.